.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "make-it-quick"]
```

Comment:
I assume this is necessary?

Reply:
Yeah, this is necessary for the action to even start in any fork of this repo. Can't really get around it.